packages/api/src/external/index.ts

```diff
@@ -8,8 +8,6 @@ export function isMedicalDataSource(s?: string | null): s is MedicalDataSource {
   return list.includes(s);
 }
 
-export const HL7OID = "2.16.840.1.113883";
```

Comment:
nit: we could keep this and use to define the CW str below `"2.16.840.1.113883.3.3330"` -> adds a bit of context as to how the OID is structured